extensions/multisuite/suite.go

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2021 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package multisuite
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/networkservicemesh/gotestmd/pkg/suites/shell"
+)
+
+type Suite struct {
+	shell.Suite
+
+	suits []interface{}
```

Comment:
```suggestion
	suits []suite.TestingSuite
```

Reply:
This array contains interfaces of different types, such as: `suite.TestingSuite`, `suite.SetupAllSuite`, `suite.Setup`, `suite.TearDownAllSuite`, `suite.TearDownTestSuite`, `suite.BeforeTest`, `suite.AfterTest`.

Reply:
see `multisuite` implementation for clarifying. Subsuite can implements only one interface from the list above.

Reply:
yes, but we can use `suite.TestingSuite` instead of `interface`...

Reply:
Replaced